packages/application-shell/src/apollo-links/header-link.ts

```diff
@@ -70,6 +71,8 @@ const headerLink = new ApolloLink((operation, forward) => {
       'X-Team-Id': teamId,
       'X-Application-Id': window.app.applicationId,
       'X-Feature-Flag': featureFlag,
+      // Merge additional headers that may have been set in the `context.headers` option.
+      ...additionalHeaders,
```

Comment:
Can we explicitely add the `x-forward-to`? I would avoid spreading as it gives the impression that the MC API supports all headers which it doesn't. 

I understand that we want to support any header to other services (`uri`s) but that's separate to me. We could either 

1. Optimistically spread them
2. Add a `allowAnyHEader` property
3. Evaluate the `uri` to not be one of ours

Just a thought but I think spreading them (as in 1.) is a bit dangerous.

Reply:
Hmm it‘s not just one header.

Let me think about it, thanks for raising this up